Examples/Scripts/compareRootFiles.hpp

```diff
@@ -451,20 +469,34 @@ struct BranchComparisonHarness {
     CREATE_VECTOR__HANDLE_TYPE(bool)
 
     // Handle vectors of all standard floating-point types
-    else CREATE_VECTOR__HANDLE_TYPE(float) else CREATE_VECTOR__HANDLE_TYPE(
-        double)
-
+    else CREATE_VECTOR__HANDLE_TYPE(float)       //
+        else CREATE_VECTOR__HANDLE_TYPE(double)  //
```

Comment:
```suggestion
    else CREATE_VECTOR__HANDLE_TYPE(float)   //
    else CREATE_VECTOR__HANDLE_TYPE(double)  //
```

Reply:
I can disable clang-format and see if I can improve the format.